src/transform/adm0_surround.py

```diff
@@ -0,0 +1,70 @@
+#############################################################################
+# Process Surrounding Countries for GADM AOI.
+#############################################################################
+
+import sys
+import os
+import fiona
+import geopandas as gpd
+import logging
+import datetime
+import pandas as pd
+from jsonschema import validate
+
+from utils.yaml_api import parse_yaml 
+from utils import gadm_gpkg_processing
+
+import pdb
+
+##
+#
+def transform():
```

Comment:
This method should take the parameters passed via the snakefile as input parameters